utils/utils.go

```diff
@@ -60,9 +61,40 @@ func Copy(source, destination string) error {
 	return os.WriteFile(destination, input, 0600)
 }
 
-// Move moves the file to destination.
+// Move moves the file at source to destination.
 func Move(source, destination string) error {
-	return os.Rename(source, destination)
+	err := os.Rename(source, destination)
+	if err != nil {
+		log.Warnln("Moving with os.Rename failed, falling back to copy and delete!", err)
+		return moveFallback(source, destination)
+	}
+	return nil
```

Comment:
Only warn once?

Reply:
No, `moveFallback(source, destination)` does the copy and delete.

Reply:
I don't get what you mean. It has the same logging frequency as my error case in #1768 (one per failed move).